test/e2e-tests.sh

```diff
@@ -44,6 +44,7 @@ fi
 
 initOS
 install_kustomize
+npm install -g newman
```

Comment:
we should pin to a specific version or at least the current major version to avoid any nasty surprises with semver breaking changes:

`npm install -g newman@5`

Reply:
fixed